src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -106,6 +106,39 @@ class CommonUtils {
     }
 
     /**
+     * Utility to add a performance publisher block, for use in performance tests
+     *
+     * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">Common utils</a>
+     */
+
+    static void addPerformancePublisher(context, String reportPattern="**/results/*.jtl", String unstableResponseTimeThreshold="", int failedThresholdPositive, int failedThresholdNegative, int unstableThresholdPositive, int unstableThresholdNegative) {
+        context.with {
+            configure {
+                it / publishers << 'hudson.plugins.performance.PerformancePublisher' {
```

Comment:
Is the convention here to use '<<' or '/'?  I think this will always append a new performance publisher node in the xml.  I'm not sure what happens if there are two such blocks, but it might be better to use '/' here.